stackslib/src/clarity_vm/tests/events.rs

```diff
@@ -279,7 +279,7 @@ fn test_emit_nested_print_nok() {
                 (err u1)))
         (define-public (emit-event-ok)
             (begin
-                (emit-event-nok)
+                (unwrap-err! (emit-event-nok) (err u1))
```

Comment:
Ok. This is a small test. And this change will work both in wasm and interpreter runtimes without change the test nature/assertion. 👍